Patterns/French/French-NumbersWithUnit.yaml

```diff
@@ -723,7 +723,7 @@ BuildPrefix: !simpleRegex
   def: (?<=(\s|^|\P{L}))
 BuildSuffix: !simpleRegex
   def: (?=(\s|\P{L}|$))
-ConnectorToken: de
+ConnectorToken: 'de '
```

Comment:
Why add this whitespace?

Reply:
because otherwise 'de' in 'degres' is interpreted as connector and it fails to resolve the unit 'gres'.

Reply:
Could you spend some time trying to find a better solution? It seems not a good choice to add an arbritary whitespace here for one language and also pay the cost of trimming thousand of times across all cultures.
At the very least, there should be a comment in the YAML about it.
As the issue was not high-priority, we'll hold merging it a bit.

Reply:
ok, I will check

Reply:
Implemented new solution where the token with space is used only by the Parser. In this way there is no need to trim in the Extractor and other languages are not affected. 
Also, now Temperature supports the connector as well.